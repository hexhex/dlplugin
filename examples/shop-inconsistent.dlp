% -*- prolog -*-

minusShop("Shop","s1").

consistent :- &dlConsistent["shop.owl",plusShop,minusShop,plusProvides,foo].
