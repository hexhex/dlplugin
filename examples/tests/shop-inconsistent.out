{minusShop("shop:Shop","shop:s1"), minusShop("shop:Shop","shop:s2"), minusShop("shop:Shop","shop:s3"), minusShop("shop:Shop","shop:s4"), minusShop("shop:Shop","shop:s8"), minusShop("shop:Shop","shop:s5"), minusShop("shop:Shop","shop:s6"), minusShop("shop:Shop","shop:s7"), minusShop("shop:Shop","shop:s9"), minusShop("shop:Shop","shop:s10")}
